devsetup/scripts/standalone.sh

```diff
@@ -58,10 +58,10 @@ sudo hostnamectl set-hostname standalone.localdomain --transient
 
 export NTP_SERVER=${NTP_SERVER:-"clock.corp.redhat.com"}
 export EDPM_COMPUTE_CEPH_ENABLED=${EDPM_COMPUTE_CEPH_ENABLED:-true}
-export CEPH_ARGS=${CEPH_ARGS:-"-e ~/deployed_ceph.yaml -e /usr/share/openstack-tripleo-heat-templates/environments/cephadm/cephadm-rbd-only.yaml"}
+export CEPH_ARGS=${CEPH_ARGS:-\"-e \$HOME/deployed_ceph.yaml -e /usr/share/openstack-tripleo-heat-templates/environments/cephadm/cephadm-rbd-only.yaml\"}
```

Comment:
This didn't work for me. What worked is using single-quotes like:

```
-export CEPH_ARGS=${CEPH_ARGS:-"-e ~/deployed_ceph.yaml -e /usr/share/openstack-tripleo-heat-templates/environments/cephadm/cephadm-rbd-only.yaml"}                                    
+export CEPH_ARGS=${CEPH_ARGS:-'-e ~/deployed_ceph.yaml -e /usr/share/openstack-tripleo-heat-templates/environments/cephadm/cephadm-rbd-only.yaml'}                                    
```

Reply:
I pushed a new commit with single quotes and will test it now

Reply:
@booxter I tested this approach in https://review.rdoproject.org/zuul/build/43d3c98ea8ac4a499e8d04d0e6bf4c5c and it failed to found the `deployed_ceph.yaml`: https://logserver.rdoproject.org/16/49316/4/check/data-plane-adoption-github-rdo-centos-9-crc-single-node/43d3c98/ci-framework-data/logs/ci_make_002_run_standalone.log